<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraspPlannerEvaluator</class>
 <widget class="QMainWindow" name="GraspPlannerEvaluator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraspStudio - GraspPlanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPlan">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>183</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Plan grasp(s)</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>27</y>
         <width>181</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Grasp Planning Options</string>
       </property>
       <widget class="QCheckBox" name="checkBoxFoceClosure">
        <property name="geometry">
         <rect>
          <x>23</x>
          <y>135</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Force closure</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxGraspNumber">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>37</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of Grasps</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>58</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Timeout, seconds (0=endless)</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxTimeOut">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>76</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>97</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Min quality</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxQuality">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>115</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>241</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save XML</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>212</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>212</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>265</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>273</y>
         <width>241</width>
         <height>361</height>
        </rect>
       </property>
       <property name="title">
        <string>Pertubed Options</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>136</x>
          <y>40</y>
          <width>21</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>155</x>
          <y>62</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Out of</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonNormalGrasp">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>29</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Execute Normal 
Grasp</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxGraspNum">
        <property name="geometry">
         <rect>
          <x>154</x>
          <y>34</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelGraspNum">
        <property name="geometry">
         <rect>
          <x>204</x>
          <y>57</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>85</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonResetPose">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>299</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset Poses</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPerturbedGrasp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Perturbed Grasp</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>95</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Translation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>76</x>
          <y>125</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X axis</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPertDistanceX">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>120</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPertDistanceY">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>146</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>76</x>
          <y>150</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y axis</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPertDistanceZ">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>170</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>76</x>
          <y>172</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Z axis</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPertAngleX">
        <property name="geometry">
         <rect>
          <x>125</x>
          <y>120</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPertAngleY">
        <property name="geometry">
         <rect>
          <x>125</x>
          <y>146</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPertAngleZ">
        <property name="geometry">
         <rect>
          <x>125</x>
          <y>172</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>195</x>
          <y>173</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Z axis</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>195</x>
          <y>151</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y axis</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>126</x>
          <y>95</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>195</x>
          <y>126</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X axis</string>
        </property>
       </widget>
       <widget class="Line" name="line_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>274</y>
          <width>141</width>
          <height>30</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>238</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Vis. Delay (ms)</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxStep">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>206</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>331</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>730</y>
         <width>241</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Information</string>
       </property>
       <widget class="QLabel" name="labelInfo">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>221</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Grasps: 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>720</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>467</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxVDelay">
       <property name="geometry">
        <rect>
         <x>119</x>
         <y>505</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>619</y>
         <width>241</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Visualization Options</string>
       </property>
       <widget class="QCheckBox" name="checkBoxHighlight">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>86</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Highlight</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxCones">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>46</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Friction Cones</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxGrasps">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>66</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show grasps</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxColModel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>26</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision Model</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
